docs/getting-started/concepts.mdx

```diff
@@ -69,7 +69,7 @@ For more, visit the [Access Control documentation](/docs/access-control/overview
 </Banner>
 
 You can specify population `depth` via query parameter in the REST API and by an option in the local API. *Depth has no effect in the GraphQL API, because there, depth is based on the shape of your queries.*
-
+It is also possible to limit the depth for a specific field using the `maxDepth` property in your configuration.
```

Comment:
Let's be explicit and note that this property only applies to `relationship` and `upload` field types